frontend/talentsearch/src/js/components/skills/SkillChecklist/SkillChecklist.tsx

```diff
@@ -30,15 +38,15 @@ const Family: React.FunctionComponent<FamilyProps> = ({
       data-h2-padding="b(all, xxs)"
       key={family.key}
     >
-      <input
-        type="checkbox"
-        id={family.id}
-        onChange={(e) => {
-          callback(family, e.target.checked);
-        }}
-      />
-      &nbsp;
-      <label htmlFor={family.id}>
+      <label htmlFor={htmlForAssigner}>
+        <input
+          type="checkbox"
+          id={htmlForAssigner}
```

Comment:
I'm not clear on why this change is necessary.  Can't the id stay as `family.id`?

Reply:
jest was throwing errors about "absence of a `for`" this fixed it
I think it wanted labels and id to match

Reply:
Just using the label wrapping that you did with a plain family.id seems to be OK for me.  Are we getting different errors again?
![image](https://user-images.githubusercontent.com/8978655/165946850-7a84e39f-74b7-4ca7-a6ad-185d56bd53e2.png)

Reply:
I'd be curious to see the error since nesting an `input` within a `label` creates an association so the `htmlFor` attribute should not be needed.

Reply:
@esizer 
If I remove the `htmlFor`

![image](https://user-images.githubusercontent.com/40485260/165977891-97a0a7e2-0e25-44ff-a028-6b81bd040591.png)

Reply:
okey, done in the last commit

Reply:
Nice! 👍

Reply:
I could be wrong here but I think we should be able to remove the `id` completely since it seems it was only being used to satisfy the deprecated `jsx-a11y/label-has-for` rule.

Reply:
@esizer @petertgiles 
`id` removed